### ISSUE TITLE
ResponseContinuation keeps growing

### ISSUE THREAD

**Issue by joopscheer** [Azure/azure-documentdb-dotnet#330]
In my application I'm using the Azure DocumentDB API to step trough many records. To get the next 50 items the "RequestContinuation" of the "FeedOptions" is used. It seems that the "ResponseContinuation" token in the FeedResponse keeps growing and growing. After a few request the token is more than 2000 characters. The related issue #61 has been closed.

**Comment by kirankumarkolli** [comment]
@joopscheer yes it can grow really large. Are you running into any issues due to this?

**Comment by joopscheer** [comment]
Yes. To get to the next results the token is added to a link in my web app. The web app runs on Azure. Azure has a max url length of 2048.

**Comment by joopscheer** [comment]
@kirankumarkolli Any news regarding this issue?

**Comment by jamesthurley** [comment]
You are now able to set the `ResponseContinuationTokenLimitInKb` property on the `FeedOptions` object to limit the continuation token length.

**Comment by f1crazed** [comment]
This is an issue when trying to use a Azure Logic app. There is no way to set the ResponseContinuaionTokenLimitInKb. If the query has more than one filter in the where clauses you end up getting a 400 - Bad Request - Request Too Long response. I think someone need to rethink the implementation of the token.
